.npmrc

```diff
@@ -0,0 +1 @@
+legacy-peer-deps=true
```

Comment:
This would actually be a `build` or `ci` prefix. This update has no impact in a released version or the performance of the element to the consumer.